src/sentry/static/sentry/app/utils/replaceRouterParams.jsx

```diff
@@ -0,0 +1,18 @@
+export default function replaceRouterParams(route, params) {
+  // parse route params from route
```

Comment:
does this have tests? im reading from top to bottom so dismiss this if it does. it looks like it could hide weird bugs

Reply:
Ya it has tests